apis/clusterresources/v1beta1/awsencryptionkey_types.go

```diff
@@ -63,7 +63,7 @@ type AWSEncryptionKeyList struct {
 }
 
 func (aws *AWSEncryptionKey) GetJobID(jobName string) string {
-	return client.ObjectKeyFromObject(aws).String() + "/" + jobName
+	return aws.Kind + "/" + client.ObjectKeyFromObject(aws).String() + "/" + jobName
```

Comment:
Maybe just use UUID-Based Naming. It would be like `aws.ObjectMeta.UID + "/" + jobName`. You can even add `client.ObjectKeyFromObject(aws).String()` there. WDYT?

Reply:
The following solution allows us to easly watch in the operator logs for which resource job is created. However, we can also add UUID to the job name. What do others think about a pattern with UUID included?`metadata.uuid/metadata.kind/metadata.namespace/metadata.name/jobid`
It looks a bit big, but it provides more uniqueness for JobIDs.

Reply:
I discussed it with @ribaraka and decided to use `metadata.kind/metadata.namespace/metadata.name/jobId` because it is easier to manage jobs via the operator logs.

Reply:
Looks also good